documentation/projects/proposals/publish_dataset/20230706-project_proposal.md

```diff
@@ -0,0 +1,225 @@
+
+It is worth noting that this year we identified many projects to work on which
+rely on bulk analysis of Openverse's data. These projects could be replaced by,
+or made easier by the publication of the datasets. This could work in a few
+ways. A community member, training a model using the Openverse dataset,
+generates metadata that we want and planned to generate ourselves. Then, the
+HuggingFace platform presents an alternative to other SaSS products we intended
+to use to generate machine labels, detect sensitive content, and so on. Instead
+of those offerings we use models hosted on the HuggingFace hub. The
+[Datasets library](https://huggingface.co/docs/datasets/index) allows for easy
+loading of the Openverse dataset in any data pipelines we write. HuggingFace
+also offers the ability to deploy production-ready API endpoints for
+transformation models hosted on their hub. This feature is called
+[Inference Endpoints](https://huggingface.co/docs/inference-endpoints/index).
+
+#### Potentially-related projects
+
+- [External API for sensitive content detection #422](https://github.com/WordPress/openverse/issues/422) -
+  Instead of Rekognition or Google Vision, we may want to use a community model
+  on HuggingFace.
+- [Machine Image Labeling pipeline #426](https://github.com/WordPress/openverse/issues/426)
+  Either a community member may create an up to date set of machine-generated
+  labels we could use, or we could again create an Inference Endpoint with an
+  existing model, for example
+  [Vision Transformer](https://huggingface.co/google/vit-base-patch16-224)
+- [Duplicate image detection #427](https://github.com/WordPress/openverse/issues/427) -
+  Deduplication is an extremely common and important preprocessing step. It will
+  surely be implemented by community contributors and can be reapplied to
+  Openverse itself. It's possible we'll want a different deduplication strategy
+  that considers provenance and ownership (i.e, did the original author upload
+  or someone else), but at a minimum, we should be able to use community
+  solutions to _identify_ duplicates.
+
+### Resiliency
+
+The metadata for openly-licensed media, used by Openverse to power the API and
+frontend, is a community utility and should be available to all users,
+_distinctly_ from Openverse itself. By publishing this dataset, we ensure access
+to this data is fast, accessible, and resilient. With published datasets, this
+access remains even if Openverse is inaccessible, under attack, or experiencing
+any other unforeseen disruptions.
+
+## Goals
+
+<!-- Which yearly goal does this project advance? -->
+
+This project encompasses all of our 2023 "lighthouse goals", but "Community
+Development" is perhaps the broadest and most relevant. Others touched on here,
+or impacted through potential downstream changes, are "Result Relevancy",
+"Quantifying our Work","Search Experience", "Content Safety", and "Data
+inertia".
```

Comment:
```suggestion
"Quantifying our Work", "Search Experience", "Content Safety", and "Data
inertia."
```

Reply:
Corrected the spacing, but the period should remain outside the quote here.